lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -334,31 +458,52 @@ def __init__(self, config: PbenchServerConfig, logger: Logger, schema: Schema):
         port = config.get("elasticsearch", "port")
         self.es_url = f"http://{host}:{port}"
         self.schema = schema
+        self.role = role
 
-    def _get_user_term(self, user: str) -> dict:
+    def _get_user_term(self, json: JSON) -> Dict[str, str]:
         """
-        _get_user_term Generate the user term for an Elasticsearch document
-        query. If the specified user parameter is not None, search for
-        documents with that value as the authorized owner. If the user
-        parameter is None, instead search for documents with public access.
+        Generate the user term for an Elasticsearch document query. If the
+        specified "user" parameter is not None, search for documents with that
+        value as the authorized owner. If the "user" parameter is absent or
+        None, and the "access" parameter is "public", search for all published
+        documents.
+
+        TODO: Currently this code supports either all published data, or all
+        data owned by a specified user. More work in query construction must
+        be completed to support additional combinations. I plan to support
+        additional flexibility in a separate PR based on issue #2370.
```

Comment:
Perhaps consider re-wording the last sentence to remove the first person language?  `Support for additional flexibility tracked via issue #2370`.

Reply:
Indeed. Again, though, I'll be ripping this out, so the `TODO` is temporary, and the "I" was somewhat deliberate as I intend to do this myself.

Reply:
What was that saying about paving roads?